pytorch_lightning/utilities/memory.py

```diff
@@ -141,3 +142,24 @@ def get_model_size_mb(model: Module) -> float:
     torch.save(model.state_dict(), model_size)
     size_mb = model_size.getbuffer().nbytes / 1e6
     return size_mb
+
+
+# CPU device metrics
+_CPU_VM_PERCENT = "cpu_vm_percent"
+_CPU_PERCENT = "cpu_percent"
+_CPU_SWAP_PERCENT = "cpu_swap_percent"
+
+
+def get_cpu_process_metrics() -> Dict[str, float]:
```

Comment:
keep this inside accelerator/cpu.py?
since we have nvidia stat collection utility inside `accelerator/gpu`.

Reply:
Circular imports issue

Reply:
We can use a local import. Done